chpl/chpl-service/src/main/java/gov/healthit/chpl/util/ValidationUtils.java

```diff
@@ -137,6 +138,14 @@ public boolean hasCert(CertificationCriterion criterion, List<CertificationCrite
         return hasCert;
     }
 
+    public boolean hasAnyCriteria(List<CertificationCriterion> criteriaToFind,
+            List<CertificationCriterion> allCriteria) {
+        Optional<CertificationCriterion> foundCriterion = criteriaToFind.stream()
+                .filter(criterionToFind -> hasCriterion(criterionToFind, allCriteria))
+                .findAny();
+        return foundCriterion.isPresent();
```

Comment:
Can these 2 lines be combined?
```suggestion
        return criteriaToFind.stream()
                .filter(criterionToFind -> hasCriterion(criterionToFind, allCriteria))
                .findAny()
                .isPresent();
```